Src/Newtonsoft.Json.UnityConverters/Helpers/TypeExtensions.cs

```diff
@@ -0,0 +1,25 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+
+namespace Newtonsoft.Json.UnityConverters.Helpers
+{
+    public static class TypeExtensions
+    {
+        public static IEnumerable<Type> GetLoadableTypes(this Assembly assembly)
+        {
+            try
+            {
+                return assembly.GetTypes();
+            }
+            catch (ReflectionTypeLoadException ex)
+            {
+#if DEBUG
+                Console.WriteLine($"Newtonsoft.Json.UnityConverters.Helpers.TypeExtensions: Failed to load some types from assembly '{assembly.FullName}'. Maybe assembly is not fully loaded yet?\n{ex}");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Split this 203 characters long line (which is greater than 200 authorized).](https://app.codacy.com/manual/jilleJr/Newtonsoft.Json-for-Unity.Converters/pullRequest?prid=5459451)